syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -413,7 +413,7 @@ def matmul(self, *args, **kwargs):
             ), "In matmul, all args should have the same precision_fractional"
 
         if isinstance(self.child, AdditiveSharingTensor) and isinstance(other.child, torch.Tensor):
-            # If we try to matmul a FPT>(wrap)>AST with a FPT>torch.tensor,
+            # If we try to matmul a FPT>(wrap)>AST with a FPT>(wrap)>torch.tensor,
```

Comment:
Nope, `FPT>(wrap)>torch.tensor` is forbidden!

But actually `FPT>(wrap)>AST` is not correct anymore, now we do `FPT>AST`  directly